test/test_runner.sh

```diff
@@ -27,10 +27,13 @@ function execute() {
 }
 
 # Tests to run. Default is "." (i.e. the current directory).
-TESTS=("${@:-.}")
+TESTS_ROOT=("${@:-.}")
+# Serial tests
+SERIAL_TESTS=("${TESTS_ROOT[@]}/serial")
 
-# The number of parallel jobs to execute
+# The number of parallel jobs to execute non-serial tests
 export JOBS=${JOBS:-$(($(nproc --all) * 4))}
 
 # Run the tests.
-execute bats --jobs "$JOBS" --tap "${TESTS[@]}"
+bats --jobs "$JOBS" --tap "${TESTS_ROOT[@]}"
+bats --tap "${SERIAL_TESTS[@]}"
```

Comment:
I think the return value of test_runner.sh should take into account the result of both test sets ... 

somethink like
```suggestion
# Run the tests.
bats --jobs "$JOBS" --tap "${TESTS_ROOT[@]}"
parallel_test_result=$?
bats --tap "${SERIAL_TESTS[@]}"
serial_test_result=$?

[ parallel_test_result && serial_test_result ]
```